packages/editor/src/components/collab-sidebar/index.js

```diff
@@ -288,7 +320,18 @@ export default function CollabSidebar() {
 			}
 		} );
 
-		return result;
+		const blockCommentIds = getCommentIdsFromBlocks();
+
+		const uniqueIds = [ ...new Set( blockCommentIds.values() ) ];
```

Comment:
Is the values call really needed her? Set accepts the array?

Reply:
Also, if you need unique IDs, maybe `getCommentIdsFromBlocks` can already check if it's been added before?